src/ado/reprun.ado

```diff
@@ -927,7 +933,7 @@ end
     local sl "{c |}{col `c1'}"
     local sl "`sl'{c |}{dup 6: }Seed RNG State{col `c2'}"
     local sl "`sl'{c |}{dup 6: }Sort Order RNG{col `c3'}"
-    local sl "`sl'{c |}{dup 6: }Data Signature{col `c4'}"
+    local sl "`sl'{c |}{dup 6: }Data Checksum{col `c4'}"
```

Comment:
If we are replacing data signature with data checksum, what does the column sort order RNG add?

Reply:
Tells you what the problem is